extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/scores.html

```diff
@@ -210,7 +206,11 @@
   </div>
 </div>
 </div>
-
 {/for}
+<script>
```

Comment:
@FroMage I used the method of `data` attribute to fix that Gauge Loading issue but it's not giving desired results 🤔